trivia-app/client/src/components/GameScreen.js

```diff
@@ -0,0 +1,48 @@
+import Question from "./Question";
+import EndScreen from "./EndScreen";
+import Score from "./Score";
+import { useState } from "react";
+
+const GameScreen = ({
+  data,
+  setStarted,
+  setAmount,
+  setCategory,
+  setDifficulty,
+}) => {
+  const [score, setScore] = useState(0);
+  console.log(data);
+  const [questionNum, setQuestionNumber] = useState(0);
+  const resetGame = () => {
+    setStarted(false);
+    setAmount(null);
+    setCategory(null);
+    setDifficulty(null);
+  };
+
+  return (
+    <div className="GameScreenDiv">
+      {questionNum !== data.data.results.length ? (
+        <>
+          <Question
+            amount={data.data.results.length}
```

Comment:
this is a little confusing here. we use the variable `amount` to represent state elsewhere in the code, but here we're setting it equal to a piece of api repsonse data. we don't want to find ourselves in a situation where we're  mutating state. the easiest workaround might be to change one of the variable names to avoid any accidental state conflict.